css/index.css

```diff
@@ -1 +1,513 @@
+    }
+
+    .s1 p {
+    font-size: 1.9rem;
+    width: 100.00%;
+    max-width: 325px;
+}
+
+.s2 p {
+    font-size: 1.9rem;
+    width: 100%;
+    max-width: 190px;
+}
+
+    .s1 {
+    display:flex;
+    width: 45.14%;
+    max-width: 325px;
+    flex-direction:column;
+    height: 180px;
+    align-items: space-between;
+    justify-content: space-between;
+    }
+
+    .s2 {
+        display:flex;
+        width: 26.39%;
+        max-width: 190px;
+        flex-direction:column;
+        height: 250px;
+        align-items: space-between;
+        justify-content: space-between;
+        }
+    
+        footer {
+            width: 90.00%;
+            max-width: 720px;
+        }
+}
+
+/* New Media Query */
+
+
+@media (max-width:500px) {
+
+    .Main {
+        width: 100%;
+        max-width: 500px;
+    }
+
+    .Main2 {
+        display:flex;
+        flex-direction:column;
+        align-items: center;
+        width: 100%;
+        max-width: 500px;
+    }
+
+    .Sec1 {
+        justify-content: space-between;
+        align-items: center;
+        margin-top: 140px;
+        width: 84%;
+        max-width: 420px;
+        height: 300px;
+    }
+
+    header {
+        margin: 0px;
+        flex-direction: column;
+        height: 500px;
+        width: 100%;
+        max-width: 500px;
+        justify-content: space-around;
+        align-items: center;
+    }
+
+    nav {
+        text-align: center;
+        flex-direction: column;
+        width: 100%;
+        max-width: 500px;
+        order:1;
+    }
+
+    nav a {
+        font-size: 4rem;
+        padding: 20px 4%;
+        border-top: 2px solid gray;
+
+    }
+    .Connect {
+        border-bottom: 2px solid gray;
+    }
+    
+    header img {
+        height: 100px;
+        order:0;
+        width: 100%;
+        max-width: 300px;
+        padding: 20px 4%;
+        align-self: center;
+    }
+
+    h1 {
+        font-size: 5.4rem;
+        text-align: center;
+    }
+
+    .Sec1s1 {
+        display:flex;
+        align-items: center;
+        justify-content: space-around;
+        flex-direction:column;
+        width: 100%;
+        max-width: 420px;
+        height: 300px;
+    }
+
+    .Sec1 a {
+        text-align: center;
+        font-size: 2.7rem;
+        padding: 10px 4.76%;
+        border: 2px solid gray;
+    }
+
+    .Sec2 {
+        width: 100%;
+        max-width: 500px;
+        padding: 20px 0px 60px 0px;
+    }
+
+    .Sec2 img {
+        width: 100%;
+        max-width: 500px;
+        margin: 10px 0px 40px 0px;
+       }
+
+    .Sec2s1 {
+
+        width: 84%;
+        max-width: 420px;
+        display:flex;
+        flex-direction:column;
+        height:375px;
+        justify-content: space-between;
+    }
+
+    
+    .Sec2s2 {
+        height: 525px;
+        width: 84%;
+        max-width: 420px;
+        display:flex;
+        flex-direction:column;
+        justify-content: space-between;
+    }
+
+  
+    .s1 p {
+    font-size: 14pt;
```

Comment:
Great job converting most of the font-sizes to rem! I see you are using `pt` here. May I ask why?